src/layers/models/fileValidator.ts

```diff
@@ -113,4 +141,52 @@ export class FileValidator {
     });
     return allValid;
   }
+
+  private validateGpkgIndex(files: string[], originDirectory: string): void {
+    files.forEach((file) => {
```

Comment:
add try catch to this method and in the catch throw message "failed to validate Gpkg Index"